lib/typescript.js

```diff
@@ -62,6 +62,24 @@ function convertType(swaggerType, swagger) {
                 typespec.properties.push(property);
             });
         }
+
```

Comment:
This _forEach should replace the above one, otherwise, it will result in duplicated typescript properties.